File: task11(C#)/task11/task11/Models/UserData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Newtonsoft.Json;

namespace task11.Models
{
    public class UserData
    {
        [JsonProperty("source")]
        public string Source { get; set; }
        [JsonProperty("data")]
        public User User { get; set; }
    }
}

File: task11/src/main/java/entity/UserData.java
package entity;

public class UserData {
    private String source;
    private User data;

    public UserData() {
    }

    public UserData(String source, User data) {
        this.source = source;
        this.data = data;
    }

    public String getSource() {
        return source;
    }

    public void setSource(String source) {
        this.source = source;
    }

    public User getData() {
        return data;
    }

    public void setData(User data) {
        this.data = data;
    }
}
